### ISSUE TITLE
php-cs-fixer job is failing from PRs

### ISSUE THREAD

**Issue by xHeaven** [archtechx/enums#29]
The php-cs-fixer job is failing if it runs from a remote branch and finds issues - for example, a PR with wrong indenting. See [here](https://github.com/archtechx/enums/actions/runs/12533262795/job/34952839487?pr=28).

**Comment by stancl** [comment]
Yeah, this is known. It's the same in our other repos based on our package template. Tried to solve it in the past but it was a bit painful so it's on hold now. Essentially the CI workflow passes if the user submitting the PR has write perms to the repo, not otherwise. It is still informative in terms of whether php-cs-fixer passed or not, it just doesn't auto fix PRs.
